CPAC/resources/configs/pipeline_config_template.yml

```diff
@@ -409,19 +409,22 @@ runRegisterFuncToMNI :  [1]
 
 
 # The resolution (in mm) to which the preprocessed, registered functional timeseries outputs are written into. Note that selecting a 1 mm or 2 mm resolution might substantially increase your RAM needs- these resolutions should be selected with caution. For most cases, 3 mm or 4 mm resolutions are suggested.
-resolution_for_func_preproc :  3mm
+resolution_for_func_preproc :  3 # if 3 dim different, enter: num1xnum2xnum3, eg. 3x2.5x2.5
```

Comment:
Can we put this comment in the line above, as an example?

Reply:
oh sure. have changed that